crates/topos-tce-storage/src/tests/mod.rs

```diff
@@ -354,17 +355,22 @@ async fn pending_certificate_can_be_removed(store: Arc<ValidatorStore>) {
 
     assert!(matches!(pending_column.get(&pending_id), Ok(None)));
 
-    let _ = store.insert_pending_certificate(&certificate).unwrap();
-
     let pending_id = store
         .insert_pending_certificate(&certificate)
         .unwrap()
         .unwrap();
 
+    assert!(matches!(
+        store.insert_pending_certificate(&certificate),
+        Err(StorageError::InternalStorage(
+            crate::errors::InternalStorageError::CertificateAlreadyPending
+        ))
+    ));
+
     assert!(pending_column.get(&pending_id).is_ok());
     store.delete_pending_certificate(&pending_id).unwrap();
 
-    assert!(pending_column.iter().unwrap().next().is_some());
```

Comment:
Was this test wrong before?

Reply:
Not really, it was updated regarding the add of the `AlreadyPending` variant.